scripts/ty_benchmark/src/benchmark/lsp_client.py

```diff
@@ -0,0 +1,175 @@
+"""Simple LSP client for benchmarking diagnostic response times."""
+
+from __future__ import annotations
```

Comment:
this shouldn't be required, since you set `requires-python = ">=3.14"` in the pyproject.toml file

```suggestion
```